book/src/keymap.md

```diff
@@ -257,6 +258,19 @@ This layer is a kludge of mappings, mostly pickers.
 
 > TIP: Global search displays results in a fuzzy picker, use `space + '` to bring it back up after opening a file.
 
+#### File mode
+
+This layer is a kludge of mappings, mostly pickers.
```

Comment:
This could probably be something like "a collection of file-opening commands" or something similar

Reply:
This got reverted too by the way.

Reply:
Thanks, added it back now.